src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/AvoidUninstantiatedInternalClassesTests.cs

```diff
@@ -1323,24 +1323,31 @@ internal class CSomeClass {}
                 GetCSharpResultAt(16, 16, "CSomeClass"));
         }
 
-        [Fact, WorkItem(2957, "https://github.com/dotnet/roslyn-analyzers/issues/2957")]
-        public async Task CA1812_DesignerAttributeTypeName_NoDiagnostic()
+        [Theory]
+        [WorkItem(2957, "https://github.com/dotnet/roslyn-analyzers/issues/2957")]
+        [WorkItem(1708, "https://github.com/dotnet/roslyn-analyzers/issues/1708")]
+        [InlineData("System.ComponentModel.DesignerAttribute")]
+        [InlineData("System.Diagnostics.DebuggerTypeProxyAttribute")]
+        public async Task CA1812_DesignerAttributeTypeName_NoDiagnostic(string attributeFullName)
```

Comment:
Rename the test as this is no longer specific to designer attribute? Same for other modified tests..